web/src/components/vis/AnovaTableTile.vue

```diff
@@ -89,10 +89,14 @@ vis-tile-large(v-if="plot", title="Anova Table", :loading="plot.loading", expand
     v-btn(flat, dark, block, @click="downloadTable")
       v-icon.mr-2 {{ $vuetify.icons.save }}
       | Download Table
-  anova-table(v-if="plot.data", :data="tableData", :threshold="threshold", v-model="selected")
+  anova-table.horizontalScrollable(v-if="plot.data", :data="tableData", :threshold="threshold", v-model="selected")
```

Comment:
The linter says this line is too long. The fix is to split the attributes out over several lines.

Reply:
Fixed the line that was too long.